pkg/template/registry/rest.go

```diff
@@ -120,5 +120,5 @@ func (s *REST) Create(ctx kapi.Context, obj runtime.Object) (runtime.Object, err
 	if s.legacyReturn {
 		return cfg, nil
 	}
-	return &kapi.List{Items: cfg.Items}, nil
+	return tpl, nil
```

Comment:
correct me if i'm wrong but that test should be failing spectacularly (type cast error) with this change.